ruka/ingress/lhn/ingress-s3-lhn.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
+    kubernetes.io/ingress.class: nginx-lhn
+  name: rook-ceph-rgw-ingress-lhn
+  namespace: rook-ceph
+spec:
+  rules:
+  - host: s3-lhn.dev.lsst.org
+    http:
+      paths:
+      - backend:
+          service:
+            name: rook-ceph-rgw-lfa
+            port:
+              number: 80
+        path: /
+        pathType: Prefix
+  tls:
+  - hosts:
+    - s3-lhn.dev.lsst.org
+    secretName: rook-ceph-rgw-ingress-lhn-tls
+status:
```

Comment:
AFAIK, the status hash can not be written to. Also, it isn't possible to set an IP on an ingress resource as the svc ip is a property of the ingress controller, not the ingress resource.

Reply:
so sorry i took the ingress from somewhere with -o yaml and deleted the garbage, but i missed this one!